Wireframe/css/style.css

```diff
@@ -0,0 +1,48 @@
+body {
+    font-family: Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+  header {
+    text-align: center;
+    padding: 20px;
+    background-color: #f4f4f4;
+  }
+  header h1 {
+    margin: 0;
+  }
+  header p {
+    font-style: normal;
+    font-size: x-large;
+  }
+  main {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
```

Comment:
Does the wireframe show three columns? The picture I see shows 2 columns, with an article spanning 2 columns up top and then two smaller ones underneath.